### PR TITLE
Basic configuration

### PR DESCRIPTION
**Pull request by thangqp**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    <parent>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-parent-ws</artifactId>
+        <version>20</version>
+        <relativePath/>
+    </parent>
+
+    <groupId>org.gridsuite</groupId>
+    <artifactId>gridsuite-dynamic-security-analysis-server</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
```
Comment (etiennehomer) [inline pom.xml]:
```suggestion
    <version>2.9.0-SNAPSHOT</version>
```

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+                <scope>import</scope>
+            </dependency>
+
+            <!-- project specific dependencies -->
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <!-- Compilation dependencies -->
+        <dependency>
```
Comment (etiennehomer) [inline pom.xml:103]: Use alphabetical order. com.powsybl to the top

File: src/main/java/org/gridsuite/dynamicsecurityanalysis/server/dto/DynamicSecurityAnalysisStatus.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+package org.gridsuite.dynamicsecurityanalysis.server.dto;
+
+/**
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
+public enum DynamicSecurityAnalysisStatus {
+    NOT_DONE,
+    RUNNING,
+    CONVERGED,
```
Comment (etiennehomer) [inline src/main/java/org/gridsuite/dynamicsecurityanalysis/server/dto/DynamicSecurityAnalysisStatus.java]: To be checked. Could be : 
COMPLETED,
FAILED

File: src/main/java/org/gridsuite/dynamicsecurityanalysis/server/entities/DynamicSecurityAnalysisResultEntity.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+/**
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
+@Getter
+@Setter
+@Table(name = "dynamic_security_analysis_result")
+@NoArgsConstructor
+@Entity
+public class DynamicSecurityAnalysisResultEntity implements Serializable {
```
Comment (etiennehomer) [inline src/main/java/org/gridsuite/dynamicsecurityanalysis/server/entities/DynamicSecurityAnalysisResultEntity.java]: you need " implements Serializable" ?

File: src/main/resources/application-local.yml [hunk 5]
```diff
@@ -0,0 +1,23 @@
+server:
+  port: 5040
+
+spring:
+  rabbitmq:
+    addresses: localhost
+    port: 5672
```
Comment (etiennehomer) [inline src/main/resources/application-local.yml:7]: To remove ?

File: src/main/resources/config/application.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+          destination: ${powsybl-ws.rabbitmq.destination.prefix:}dsa.cancel
+        publishStopped-out-0:
+          destination: ${powsybl-ws.rabbitmq.destination.prefix:}dsa.stopped
+        publishFailed-out-0:
+          destination: ${powsybl-ws.rabbitmq.destination.prefix:}dsa.failed
+        publishCancelFailed-out-0:
+          destination: ${powsybl-ws.rabbitmq.destination.prefix:}dsa.cancelfailed
+      output-bindings: publishRun-out-0;publishResult-out-0;publishCancel-out-0;publishStopped-out-0;publishFailed-out-0;publishCancelFailed-out-0
+
+powsybl:
```
Comment (etiennehomer) [inline src/main/resources/config/application.yaml]: We rather set this in the Service

**Review by etiennehomer — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/dynamicsecurityanalysis/server/dto/DynamicSecurityAnalysisStatus.java (hunk 3 above)
Comment (thangqp) [reply src/main/java/org/gridsuite/dynamicsecurityanalysis/server/dto/DynamicSecurityAnalysisStatus.java]: This is the status of server which includes the status of computing result, in every computing server actually has 4 such values.

Comment (etiennehomer) [reply src/main/java/org/gridsuite/dynamicsecurityanalysis/server/dto/DynamicSecurityAnalysisStatus.java]: No, we have these statuses for loadflow-server (which is fully righteous), for security-analysis-server (which is not really relevant) and for dynamic-simulation-server (I don't know if it's relevant). We have different statuses for shortcircuit, sensi, voltage-init.

Comment (etiennehomer) [reply src/main/java/org/gridsuite/dynamicsecurityanalysis/server/dto/DynamicSecurityAnalysisStatus.java]: The notion of convergence is something specific

File: src/main/java/org/gridsuite/dynamicsecurityanalysis/server/entities/DynamicSecurityAnalysisResultEntity.java (hunk 4 above)
Comment (thangqp) [reply src/main/java/org/gridsuite/dynamicsecurityanalysis/server/entities/DynamicSecurityAnalysisResultEntity.java]: YES! I don't see any case in which we need Serializable, this entity has no particular relation with others and normally endpoints do not use directly an entity. 

I will remove Serializable

File: src/main/java/org/gridsuite/dynamicsecurityanalysis/server/dto/DynamicSecurityAnalysisStatus.java (hunk 3 above)
Comment (thangqp) [reply src/main/java/org/gridsuite/dynamicsecurityanalysis/server/dto/DynamicSecurityAnalysisStatus.java]: SUCCEED and FAILED are to status used in the front-end, at moment dynamic security analysis has no specific status

Is OK with SUCCEED and FAILED @etiennehomer

File: pom.xml (hunk 2 above)
Comment (thangqp) [reply pom.xml:103]: In fact, there is an implicit criteria that I use for each section (compilation, runtime, test), i.e. from technical to functional. 
In the technical part, modules are ordered by more to less important. In the functional part, they are ordered by specific levels. Alphabetic is used as the last criteria...

File: src/main/resources/application-local.yml (hunk 5 above)
Comment (thangqp) [reply src/main/resources/application-local.yml:7]: I am not sure, should run a test in local, but I see this property is declared in powsybl-ws-commons

File: src/main/resources/config/application.yaml (hunk 6 above)
Comment (thangqp) [reply src/main/resources/config/application.yaml]: Do you mean? 

powsybl:
  services:
    network-store-server:
      preloading-strategy: COLLECTION

Comment (etiennehomer) [reply src/main/resources/config/application.yaml]: https://github.com/gridsuite/loadflow-server/blob/b3998ffdafee0ef547ab665a00459b634d951163/src/main/java/org/gridsuite/loadflow/server/service/LoadFlowWorkerService.java#L125

Comment (etiennehomer) [reply src/main/resources/config/application.yaml]: Like this

File: src/main/resources/application-local.yml (hunk 5 above)
Comment (etiennehomer) [reply src/main/resources/application-local.yml:7]: Yes, we get it from powsybl-ws-commons

File: src/main/resources/config/application.yaml (hunk 6 above)
Comment (thangqp) [reply src/main/resources/config/application.yaml]: OK

File: src/main/resources/application-local.yml (hunk 5 above)
Comment (thangqp) [reply src/main/resources/application-local.yml:7]: tested, we can remove only port: 5672 as it is the same default port, addresses is used for local test

File: src/main/resources/config/application.yaml (hunk 6 above)
Comment (thangqp) [reply src/main/resources/config/application.yaml]: DONE in https://github.com/gridsuite/dynamic-security-analysis-server/pull/1/commits/9433ce504076cf82eeab5658042569ef1619363c

File: src/main/resources/application-local.yml (hunk 5 above)
Comment (thangqp) [reply src/main/resources/application-local.yml:7]: DONE in https://github.com/gridsuite/dynamic-security-analysis-server/pull/1/commits/9433ce504076cf82eeab5658042569ef1619363c

File: src/main/java/org/gridsuite/dynamicsecurityanalysis/server/entities/DynamicSecurityAnalysisResultEntity.java (hunk 4 above)
Comment (thangqp) [reply src/main/java/org/gridsuite/dynamicsecurityanalysis/server/entities/DynamicSecurityAnalysisResultEntity.java]: DONE in https://github.com/gridsuite/dynamic-security-analysis-server/pull/1/commits/9433ce504076cf82eeab5658042569ef1619363c

File: src/main/java/org/gridsuite/dynamicsecurityanalysis/server/dto/DynamicSecurityAnalysisStatus.java (hunk 3 above)
Comment (thangqp) [reply src/main/java/org/gridsuite/dynamicsecurityanalysis/server/dto/DynamicSecurityAnalysisStatus.java]: DONE in https://github.com/gridsuite/dynamic-security-analysis-server/pull/1/commits/9433ce504076cf82eeab5658042569ef1619363c

File: pom.xml (hunk 1 above)
Comment (thangqp) [reply pom.xml]: DONE in https://github.com/gridsuite/dynamic-security-analysis-server/pull/1/commits/9433ce504076cf82eeab5658042569ef1619363c

**Review by etiennehomer — APPROVED** [review]

**Pull request merged**